test_data/create_environment.sh

```diff
@@ -27,12 +27,14 @@ export DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 docker run \
```

Comment:
add `docker pull` here to ensure that you're using the latest version

Reply:
done